Example/EpoxyExample/Views/LayoutGroups/DynamicRow.swift

```diff
@@ -0,0 +1,105 @@
+// Created by Tyler Hedrick on 11/11/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import UIKit
+
+final class DynamicRow: BaseRow, EpoxyableView {
+
+  // MARK: Lifecycle
+
+  override init() {
+    super.init()
+    layout.install(in: self)
+    layout.constrainToMarginsWithHighPriorityBottom()
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  // MARK: ContentConfigurableView
+
+  struct Content: Equatable {
+    let title: String
+    let subtitle: String
+    let revealOptionsButton: String?
+    let options: [String]?
+    let footer: String
+  }
+
+  // MARK: BehaviorsConfigurableView
+
+  struct Behaviors {
+    let didTapRevealOptions: (() -> Void)?
+    let didTapOption: ((String) -> Void)?
+  }
+
+  func setContent(_ content: Content, animated: Bool) {
+    var items: [GroupItemModeling] = [
```

Comment:
unfortunately function builders don't support for loops like result builders do :/ I wasn't sure how to solve that issue so I just did it the old fashioned way